articles/user-guide/automated-ip-warmup.md

```diff
@@ -0,0 +1,68 @@
+
+**A customer with inconsistent sending patterns or concerns with specific receiving domains**
+If your team doesn’t send every day or has large bursts of sending only a couple times per day then automating the warm-up process may not be the best option for you. In order to help protect our customer’s deliverability and sending reputation we establish a daily limit during warm-up but we also enforce hourly and minute limits to smooth the flow of traffic directed towards a receiving domain. This means that consistent sending throughout each day is the best way to ensure your warm-up process maintains high deliverability and you build a positive sending reputation. If you’re sending is sporadic (one or two campaigns per week) or your subscriber base is relatively unengaged, you may not be able to warm up your new IP automatically as effectively as you could with an alternative strategy. We recommend sending to clean lists of engaged subscribers to build trust and establish a positive sending relationship on new IPs. 
+
+# If Automating isn’t Right, What is? 
+If you fall into the fourth category mentioned above or have other specific needs and concerns that aren’t covered here - that’s ok! Our incredibly dedicated team Customer Success Team or your personal Technical Account Manager can advise on the best course of action. Do some initial homework by visiting the [original IP Warm-up Knowledge Article](https://www.sparkpost.com/docs/deliverability/ip-warm-up-overview/) and get in touch!
+
+
+# Nuts and Bolts
+When you enable AutoIP Warm Up for an IP address (enabled automatically for newly purchased or assigned IPs) the stage of that IP will be set to Stage 1. Once you start sending traffic through the IP pool that contains the IP your warming up, we’ll include your cold IPs in the sending selection process up to the maximum number of emails for that Stage. Each day, the app checks to see what IPs have delivered at least 90% of the stage volume and will progress the successful IPs to the next stage. The day after those IPs meet the sending requirements for the current stage, they’ll be eligible to send a higher volume of traffic.
```

Comment:
First stab at refactoring this paragraph:

Auto IP Warm up works by guiding (better word here?) new IPs through several reputation stages until they are ready for regular email traffic. Each stage limits the number of emails that can be sent through that IP in a day. Once an IP sends enough, it will automatically be upgraded to the next stage. Any email traffic directed to an IP that has reached the stage limit will be redirected to the appropriate overflow pool. 

When you enable Auto IP Warm Up for an IP address (enabled automatically for newly purchased or assigned IPs) the IP will be set to Stage 1. Once you start sending traffic through the IP pool that contains the IP your warming up, we’ll include the cold IP in the sending selection process up to the maximum number of emails for that stage. Once the IP has sent at least 90% of the current stage volume, it’s ready to progress to the next stage. The IP is then automatically upgraded to the next stage the day after it meets the criteria.